pkg/lightning/backend/local/region_job.go

```diff
@@ -314,6 +314,21 @@ func (local *Backend) writeToTiKV(ctx context.Context, j *regionJob) error {
 	return err
 }
 
+func newWriteRequest(meta *sst.SSTMeta, ResourceGroupName, TaskType string) *sst.WriteRequest {
+	return &sst.WriteRequest{
+		Chunk: &sst.WriteRequest_Meta{
+			Meta: meta,
+		},
+		Context: &kvrpcpb.Context{
+			ResourceControlContext: &kvrpcpb.ResourceControlContext{
+				ResourceGroupName: ResourceGroupName,
+			},
+			RequestSource: util.BuildRequestSource(true, kv.InternalTxnLightning, TaskType),
+			TxnSource:     kv.LightningPhysicalImportTxnSource,
```

Comment:
please post manual test result. I'm not sure TiKV import service will use this value.

And maybe we can add a SST table / block property to skip scanning the whole SST to reduce IO.

Reply:
> maybe we can add a SST table / block property to skip scanning the whole SST to reduce IO.

It sounds great to do so, but looks more complicated since we have to consider the case such as region merge or split? Also, is it possible that there is one sst that contains lightning physical imported data and normal inserted data at the same time?

Reply:
> please post manual test result. I'm not sure TiKV import service will use this value.

Yes, this PR should be get merged after test, I will post the manual test result. 

https://github.com/tikv/tikv/pull/17895/files This PR set the `txn_source` to initialize the `txn_sst_writer`

Reply:
> > maybe we can add a SST table / block property to skip scanning the whole SST to reduce IO.
> 
> It sounds great to do so, but looks more complicated since we have to consider the case such as region merge or split? Also, is it possible that there is one sst that contains lightning physical imported data and normal inserted data at the same time?

Yes, just as a fast path. If SST merging happened we have to scan all KV.